keadm/cmd/keadm/app/cmd/helm/cloudcore.go

```diff
@@ -0,0 +1,250 @@
+/*
+Copyright 2019 The KubeEdge Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package helm
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/blang/semver"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/kubeedge/kubeedge/common/constants"
+	types "github.com/kubeedge/kubeedge/keadm/cmd/keadm/app/cmd/common"
+	"github.com/kubeedge/kubeedge/keadm/cmd/keadm/app/cmd/util"
+	kecharts "github.com/kubeedge/kubeedge/manifests"
+)
+
+const (
+	CloudCoreHelmComponent = "cloudcore"
```

Comment:
It seems that this CloudCoreHelmComponent is an internal constant, please change it to cloudCoreHelmComponent. In addition, global constants should be placed in the **api** folder.

Check other constants as well.

Reply:
Done, thanks